src/main/java/com/ss/lms/controller/BorrowerController.java

```diff
@@ -22,8 +28,9 @@
 	@Autowired
 	BorrowerService borrowerService;
 
-	@RequestMapping(path = "/api/books/checkout", method = RequestMethod.POST, headers = {
-			"Accept=application/json,application/xml" }, produces = { "application/json", "application/xml" })
```

Comment:
Is the difference here that the original one was only accepting headers that had "accept=app....." and the new version declares what you method will take